website/data/docs-navigation.js

```diff
@@ -297,7 +297,6 @@ export default [
       'docker-push',
       'docker-save',
       'docker-tag',
-      'exoscale-import',
```

Comment:
TODO, probably in the future: import docs using https://github.com/hashicorp/packer/pull/10656

Reply:
I added the TODO to the PR desc.